packages/prometheus/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.0"
```

```suggestion
- version: "1.14.1"
```
As this is a minor fix in documentation bumping the patch version would suffice.